Packs/RSANetWitness_v11_1/Integrations/RSANetWitnessv115/RSANetWitnessv115.py

```diff
@@ -1144,7 +1178,7 @@ def main() -> None:
     demisto.debug(f'Command being called is {command}')
 
     try:
-        requests.packages.urllib3.disable_warnings()
+        disable_warnings()
```

Comment:
we usually call this at the very top of the file, just after importing it, let's move it there.